packages/frint/src/createApp.ts

```diff
@@ -1,6 +1,6 @@
-import merge from 'lodash/merge';
+import { merge } from 'lodash';
```

Comment:
we have an ESLint rule for avoiding importing for `lodash` and `rxjs` directly here: https://github.com/frintjs/frint/blob/master/.eslintrc#L10

may be we can do something similar with TSLint too?